CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Node: Added ZRANGE command ([#1115](https://github.com/aws/glide-for-redis/pull/1115))
 * Python: Added RENAME command ([#1252](https://github.com/aws/glide-for-redis/pull/1252))
 * Python: Added APPEND command ([#1152](https://github.com/aws/glide-for-redis/pull/1152))
+* Node: Add `PFADD` command (TODO PR #)
```

Comment:
nit: wondering if we should just follow the same format as the rest of the entries above
```suggestion
* Node: Added PFADD command (TODO PR #)
```

Reply:
What do we write here? A feature description of PR title 1:1?

Reply:
See my comment [here](https://github.com/Bit-Quill/glide-for-redis/pull/229#discussion_r1571462767), the entry does not need to match the PR title